ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -67,7 +72,8 @@ public QueryDescription(
       @JsonProperty("ksqlHostQueryStatus") final Map<KsqlHostInfoEntity, KsqlQueryStatus>
           ksqlHostQueryStatus,
       @JsonProperty("queryType") final KsqlQueryType queryType,
-      @JsonProperty("queryErrors") final List<QueryError> queryErrors
+      @JsonProperty("queryErrors") final List<QueryError> queryErrors,
+      @JsonProperty("tasksMetadata") final Set<TaskMetadata> tasksMetadata
```

Comment:
It is having trouble Serializing `TaskMetadata` because it can't find the getters because of AK's naming conventions. I can fix this by adding `.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)` either at the mapper or just in `Console.java` but either risks exposing private fields. WDYT @abbccdda @spena ?

Reply:
It looks like all other serialization methods are depending on Ksql internally defined classes. Maybe we could define a class like `StreamTaskMetadata` to translate the original `TaskMetadata` into a class having getters.

Reply:
+1 for defining our own API model. Can you put this in `ksqldb-reset-model`?